### PR TITLE
Fixed issues found when upgrading to TypeScript 2.3.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
This meant I needed to upgrade to `@types/react-native@0.43.9`, but it looks like you've already moved onto `0.43.7` anyhow.

### CODE REVIEW

File: src/lib/containers/home.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class Home extends React.Component<Props, State> {
   render() {
     return (
       <ListView dataSource={this.state.dataSource}
-                renderScrollComponent={(props: ScrollViewStatic) => {
+                renderScrollComponent={(props: ScrollViewProps) => {
```
Comment (DanielRosenwasser) [inline src/lib/containers/home.tsx]: I believe I can remove the annotation here if you'd like.

**Comment by orta** [comment]
Awesome, thanks! 

This looks good to me, it even works with 2.2 which is what we're running with now in CI. 

We don't expect tests for OSS contributors to pass yet, so this is 👍

File: src/lib/components/text/headline.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import * as React from "react"
-import { StyleSheet, Text, TextProperties, TextStatic } from "react-native"
+import { StyleSheet, Text, TextProperties } from "react-native"
 
 export default class Headline extends React.Component<TextProperties, any> {
   render() {
     const content = (this.props.children || "") as string
     return (
-      <Text {...this.props as TextStatic} style={[styles.default, this.props.style, styles.required]} >
```
Comment (alloy) [inline src/lib/components/text/headline.tsx]: YES! I’m so glad to get rid of these things that felt kinda wrong.

**Comment by alloy** [comment]
Thanks @DanielRosenwasser! 🙏

**Pull request merged**